dot-gen/src/main/java/com/wipro/ats/bdre/ldg/GetDotForTable.java

```diff
@@ -0,0 +1,132 @@
+import com.wipro.ats.bdre.GetNodeIdForLineageRelation;
+import com.wipro.ats.bdre.lineage.LineageMain;
+import com.wipro.ats.bdre.md.dao.jpa.LineageNode;
+import com.wipro.ats.bdre.md.dao.jpa.LineageRelation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by AshutoshRai on 1/20/16.
+ */
+public class GetDotForTable {
+    protected static final Logger logger = LoggerFactory.getLogger(GetDotForTable.class);
+    private String dotString;
+
+    //fn() to get DOT through Column name & table name
+    public String dotGeneratorWithCol(String args[]) {
+        if(args.length !=2) {
+            logger.info("Two arguments required: please Give both Column name and Table name");
+            System.exit(-1);
+        }
+        GetDotForTable getDotForTable = new GetDotForTable();
+        GetNodeIdForLineageRelation getNodeIdForLineageRelation = new GetNodeIdForLineageRelation();
+
+        List<LineageRelation> lineageRelationList;
+        List<LineageRelation> lineageRelationListForTargetNodes;
+
+        //get the nodes which have the same DISPLAY_NAME as that of the string(COLUMN NAME) provided
+
+        GetLineageNodeByColName getLineageNodeByColName;
+        LineageMain lineageMain = new LineageMain();
+        //src table name is provided by user
+        String srcTableName = args[1];
+        logger.info("Generating DOT for the given Column Name: " + args[0]);
+        String targetTableName = null;
+        StringBuffer relationDot = new StringBuffer("");
+        getLineageNodeByColName = new GetLineageNodeByColName();
+        //for each relation a new node is created in LN table, so get all those
+        List<LineageNode> srcTableNodeList = getLineageNodeByColName.getTableDotFromTableName(srcTableName);
+        List<LineageNode> lineageNodeList = new ArrayList<LineageNode>();
+        int counter =0;
+        //now get all those col nodes corresponding to the tables created for each relation
+        for(LineageNode lnodes:srcTableNodeList) {
+            logger.info("----------------------------------src table" + lnodes.getDisplayName());
+             lineageNodeList.add(getDotForTable.getLineageNodeNodeId(args[0], lnodes));
+            logger.info("----------------------------------col node" + lineageNodeList.get(counter++).getDisplayName());
+        }
+        StringBuffer targetTableNode = new StringBuffer("");
+
+        //compare the list of Nodes got from LN table with LR table to get the required node
+        for (LineageNode lineageNode : lineageNodeList) {
+            //this only checks the case where the node is src
+            logger.info("------------------------inside");
+            lineageRelationList = getNodeIdForLineageRelation.execute(lineageNode.getNodeId());
+            for(LineageRelation lineageRelation:lineageRelationList) {
+            //if (lineageRelationList.size() != 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)